src/EventListener/IsGrantedListener.php

```diff
@@ -73,7 +73,7 @@ public function onKernelControllerArguments(KernelEvent $event)
                 }
             }
 
-            if (!$this->authChecker->isGranted($configuration->getAttributes(), $subject)) {
+            if (!$this->authChecker->isGranted($configuration->getAttribute(), $subject)) {
```

Comment:
If you're passing an array of attributes and are using Symfony 5, you will see the deprecation from this bundle, but the feature won't work correctly because Symfony 5 doesn't support an array of attributes, right?

I think we need to either:

A) "Mimic" the "or" behavior here - i.e. if it's an array, we call `isGranted()` twice in an "or" kind of way
B) Throw an actual exception if you're using Symfony 5 with this bundle and are passing an array of attributes. That's a BC Break, but an obvious one (better than the silent one of it just not working) AND the user *should* have seen deprecation warnings on Symfony 4.4 anyways. I think this would prevent WTF's like this one: https://github.com/sensiolabs/SensioFrameworkExtraBundle/issues/645#issuecomment-569914831